1_OCaml/TP/TP4.ipynb

```diff
@@ -19,7 +19,9 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.                    let list = [e]::listinter in
</pre><p>Ce n'est pas très judicieux de nommer une variable list vu que le mot-clef est déjà utilisé dans caml</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>